blocks/HARDWARE/ROBOTICS/ARMS/MECADEMIC/CALCULATE_CIRCLE_MOVE/CALCULATE_CIRCLE_MOVE.py

```diff
@@ -0,0 +1,55 @@
+import pandas as pd
+from typing import Optional
+from flojoy import flojoy, DataFrame
+from PYTHON.utils.mecademic_state.mecademic_calculations import getCirclePositions
+from PYTHON.utils.mecademic_state.mecademic_helpers import safe_robot_operation
+
+
+@safe_robot_operation
+@flojoy(deps={"mecademicpy": "1.4.0"})
+def CALCULATE_CIRCLE_MOVE(
+    center_X: Optional[float] = 0.0,
+    center_Y: Optional[float] = 0.0,
+    center_Z: Optional[float] = 0.0,
+    radius: Optional[float] = 0.0,
+    revolutions: Optional[float] = 1.0,
+    point_duration: Optional[int] = 500,
```

Comment:
There's no need to specify optional for all of these, having a default value is enough